ProteoformSuiteInternal/ComponentReader.cs

```diff
@@ -4,17 +4,19 @@
 using System;
 using System.Linq;
 using System.Threading.Tasks;
+using IO.Thermo;
 
 namespace ProteoformSuiteInternal
 {
     public class ComponentReader
     {
         private List<Component> raw_components_in_file = new List<Component>();
         private static List<NeuCodePair> neucodePairs_in_file = new List<NeuCodePair>();
+        private List<int> MS1_scans = new List<int>();
         public List<Component> final_components = new List<Component>();
         public HashSet<string> scan_ranges = new HashSet<string>();
 
-        public List<Component> read_components_from_xlsx(InputFile file, IEnumerable<Correction> correctionFactors)
+        public List<Component> read_components_from_xlsx(InputFile file)
```

Comment:
We'll talk next week. I'm thinking of running all of his excel files through my branch, which outputs the excel files calibrated and he wouldn't need the .txt files anymore. We're not using lock mass in the buffers anymore so no one else will need it then.